explainaboard/loaders/file_loader.py

```diff
@@ -62,6 +62,7 @@ class FileLoaderField:
     dtype: Optional[DType] = None
     strip_before_parsing: Optional[bool] = None
     parser: Optional[Callable[[str], str]] = None
+    optional: Optional[bool] = None
```

Comment:
Is there any reason to keep this value `Optional`? It looks this value is used only as `bool`: the value `None` looks meaningless.

Reply:
`optional: bool = False` needs to be put before all `Optional` variables. Since some FileLoaderField definitions specify dtype without `dtype=`, I think it can be put at the end similar to `strip_before_parsing`.

Reply:
Requirements around the order of members is whether it has the default value or not. `optional: bool = False` correctly sets the default value to the member and it can be put after `Optional` members.
Note that `Optional[T] == T | None` only means that the member becomes either a value of `T` or `None`. Defaulting to `None` does not mean "unset".

Actually, all mypy/flake8/black passes the dataclass with defaults after Optional:
```shell
$ cat temp.py                                    
from dataclasses import dataclass
@dataclass
class Foo:
    a: int
    b: int | None = None
    c: int = 42

$ mypy temp.py && flake8 temp.py && black temp.py
Success: no issues found in 1 source file
All done! ✨ 🍰 ✨
1 file left unchanged.
```

Reply:
I see. Thanks!